src/documents/Priorities.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Priorities
+layout: default
+category: API
+---
+
+Many elements of an Infusion application (in particular, [Listeners](InfusionEventSystem.md#registering-a-listener-to-an-event), 
+[Model Listeners](ChangeApplierAPI.md#model-listener-declaration), [Options Distributions](IoCSS.md) and [Subcomponents](SubcomponentDeclaration.md) ) 
+allow the use of a standardised entry named ``priority`` which allows the position of that element within a list of such elements to be adjusted.
+Depending on what the element is, the effect of the priority will be different &#8212; for example, a listener may be notified earlier or later than another
+listener, or a set of options or grades may take priority over another during options merging.
+
+The priority system is designed to be "open" in the face of evolving application designs that are worked on by different groups. As well as standard
+fixed priorities (represented by numbers or _extremal values_ such as `"first"` or `"last"`), Infusion priorities may be _constraint-based_ in which
+an element defines its priority only with respect to another element which is identified by _namespace_. These constraint-based priorities are
```

Comment:
Either here or in the table below, the word "namespace" should be explained, or be a link to a page that explains what we mean.